Re-write discussion

kcranston opened the pull request:
This is a complete re-write of the discussion section. I've moved all of the other text to the removed-text file.

kcranston commented:
If you would rather this PR be against a different branch, let me know @jar398

jar398 commented on doc/method/discussion.md:
```diff
@@ -1,250 +1,49 @@
+retained from sources and presented as part of the synthetic taxonomy. This type
+of synthesis requires that source taxonomies be available online, either through
+APIs or by bulk download in a format that can be easily parsed, and without
+terms of use that prevent such synthesis.    
```
I think "what are the shortcomings and ways it could be improved" is a question many readers will have. And talking about it will forestall criticisms that we are overselling the taxonomy (which could arise, even if they are false). It would be nice if the discussion section addressed this.

kcranston commented:
Thanks for the review! I'll make the changes in new commits on this PR.

kcranston commented:
I've addressed all of the comments except for the last one (the request for a section describing shortcomings and possible improvements). Can I ask you to tackle that in a separate PR?

jar398 commented:
merged and committed using command line.
